sandbox/sandbox_units_suffixes.py

```diff
@@ -0,0 +1,320 @@
+    'A',
+    'A\\: peak',
+    'ADC',
+    'Amps',
+    'amu',
+    'Ap',
+    'apartments',
+    'atm',
+    'atoms',
+    'bottles',
+    'boxes',
+    'C',
+    'cans',
+    'carbon\\: atoms',
+    'cases',
+    'cfm',
+    'cfs',
+    'cm',
+    'Coulumbs',
+    'cubic\\: inches',
+    'customers\\: will\\: order\\: asparagus',
+    'customers\\: will\\: order\\: fresh\\: fruit',
+    'customers\\: will\\: order\\: lemon-pepper\\: chicken',
+    'days',
+    'dB',
+    'dBV',
+    'deg',
+    'deg\\: \\frac{C}{W}',
+    'deg\\: C',
+    'degC',
+    'degC/W',
+    'Degrees',
+    'degrees\\: Celsius',
+    'dollar',
+    'dollars',
+    'E-9',
+    'E3',
+    'electrons',
+    'eV',
+    'F',
+    'Farads',
+    'fF',
+    'ft',
+    'ft.lbf',
+    'ft/(cfm)^{2}',
+    'ft/s',
+    'ft^{3}/s',
+    'ft3/s',
+    'g',
+    'g/mol',
+    'giga',
+    'gpm',
+    'grams',
+    'Gs',
+    'H',
+    # 'H\\: \\: \\: \\: \\: \\left(copy\\: from\\: Activity\\: 1\\right)',
+    # 'H\\: \\left(copy\\: from\\: Activity\\: 1\\right)',
+    'hours',
+    'Hp',
+    'hr^{-1}',
+    'hrs',
+    'hydrogen\\: atoms',
+    'Hz',
+    # 'Hz\\: \\left(10\\%\\: tolerance\\: for\\: this\\: answer\\right)',
+    'in',
+    'in.\\: of\\: water\\: \\left(gage\\right)',
+    'in^{2}',
+    'in^{3}',
+    'individuals',
+    'J',
+    'K',
+    'k\\Omega ',
+    # 'k\\Omega ,\\: tolerance\\: \\pm 5\\%',
+    'kcal',
+    'kg',
+    'kg/kmole',
+    'kHz',
+    'kibit',
+    'kJ',
+    'kJ/mol',
+    'kN',
+    'kN.m',
+    'kN/m',
+    'knot',
+    'kohms',
+    'KPa',
+    'kPa\\: \\left(abs\\right)',
+    'kPa\\: \\left(gage\\right)',
+    'ks',
+    'kW',
+    # 'kΩ',
+    'L',
+    'L/day',
+    'L/hr',
+    'lb',
+    'lb/ft2',
+    'lb\\: AP',
+    'lbf',
+    'lbf-in',
+    'lbf.ft',
+    'lbs',
+    'lone\\: pairs',
+    'm',
+    'm/s',
+    'm/s^{2}',
+    'M\\: \\: K_{2}SO_{4}',
+    'M\\: potassium\\: sulfate',
+    'M\\Omega ',
+    'm^{2}',
+    'm^{2}/s^{2}',
+    'm^{3}',
+    'm^{3}/s',
+    'm3/s',
+    'mA',
+    'mb',
+    'mcg',
+    'mCi',
+    'Megajoules',
+    'meters\\: per\\: second',
+    'meters\\: per\\: second\\: squared',
+    'MeV/nucleon',
+    'mg',
+    'mg/hr',
+    'mH',
+    'MHz',
+    'mile\\left(s\\right)\\: per\\: hour',
+    'miles\\: per\\: hour',
+    'Millivolts',
+    'mL',
+    'mL/day',
+    'mL/hr',
+    'mm',
+    'MN',
+    'Mohms',
+    'moles',
+    'mph',
+    'ms',
+    'msec',
+    'mV',
+    'mV_{p}',
+    'mV_{RMS}',
+    'mW',
+    'N',
+    'N/m^{2}',
+    # 'N\\: \\left(negative\\right)',
+    'N\\: s\\: /\\: m^{2}',
+    'N\\cdot m',
+    'N\\cdot m^{2}',
+    'neutrons',
+    'nF',
+    'nL',
+    'nm',
+    'nm^{-1}',
+    'o',
+    # 'º',
+    'ohms',
+    'on\\: eggplants\\: and\\: kiwis',
+    'Pa',
+    'pebibit',
+    'per\\: 4\\: oz\\: serving',
+    'per\\: fl\\: oz',
+    'per\\: unit',
+    'percent',
+    'pF',
+    'photons',
+    'pico',
+    'Pokeballs\\: with\\: her\\: income',
```

Comment:
oh dear, where is `pokeballs with her income` from? is this a pop culture reference I don't know or is this from a real problem?

Reply:
@lily514 lol it's from a real problem, i think asking how many pokeballs someone could buy?